src/chat-panel/components/ChatPanel.tsx

```diff
@@ -82,12 +83,19 @@ class _ChatPanel extends Component<Props, State> {
       <div
         className={this.props.activeUser!.id === message.userId ? 'messageDivActive' : 'messageDiv'}
       >
+        <div className="messageEdit" onClick={this.onMessageDeletion(message.id)}>
+          ❌✎
```

Comment:
Clicking the pencil button will delete the message.
It's better to split it out into 2 different buttons, then wrap the buttons in a containing div to position them together.